src/ui/FormWidgets/Checkbox.ts

```diff
@@ -25,6 +25,7 @@ export class Checkbox implements IFormWidgetWithLabel, IFormWidgetSelectable {
   constructor(
     public onChange: (checkbox: Checkbox) => void = (checkbox: Checkbox) => {},
     public label: string,
+    public labelSuffix?: string,
```

Comment:
For backward compatibility purpose, since this component is publicly documented, if you add the parameters at the end of the list of arguments, it would not break people already using the widget and specifying the "ariaLabel" as param #3 .

Reply:
Do you think it's a problem? I added the ariaLabel param 2 weeks ago for the same component, I would be surprised if it would be used in such a short notice

Reply:
What does the labelSuffix represent?

Reply:
Having something after the value (here it would be the count) with a different css class because it has a different style. You can see this type of prefix/suffix a lot in react-vapor